.env.ngap-lb

```
@@ -0,0 +1,94 @@
+
+# Timezone
+#TZ=Europe/Berlin
+
+# PLMN Configuration
+MCC=001
+MNC=01
+TAC=1
+
+# Docker Network
+TEST_NETWORK=172.22.0.0/24
+DOCKER_HOST_IP=192.168.1.223
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DOCKER_HOST_IP key should go before the TEST_NETWORK key

(UnorderedKey)